App/Components/Avatar.js

```diff
@@ -9,12 +9,18 @@ const Avatar = props => {
   return (
     <View style={[styles.container, style, {width, height, borderRadius}]}>
       <View style={styles.stretch}>
-        <Image
+        {uri && <Image
           source={{uri: uri}}
           resizeMode={'cover'}
           style={{width, height}}
           defaultSource={defaultSource}
-        />
+        />}
+        {/* avoids a null uri warning */}
+        {!uri && defaultSource && <Image
+          source={defaultSource}
```

Comment:
it's nice to be able to render the avatar style from a static icon sometimes (e.g. the feedview)